diff --git sources/startup.c sources/startup.c
index 83f8459..5c72735 100644
--- sources/startup.c
+++ sources/startup.c
@@ -1743,7 +1743,7 @@ int main(int argc, char **argv)
 	TimeChildren(0);
 	TimeWallClock(0);
 	PreProcessor();
-	Terminate(0);
+	//Terminate(0);
 	return(0);
 }
 /*
diff --git a/sources/unix.h b/sources/unix.h
index 0bd75a0..5efac68 100644
--- sources/unix.h
+++ sources/unix.h
@@ -33,8 +33,8 @@
 #define LINEFEED '\n'
 #define CARRIAGERETURN 0x0D
 
-#define WITHPIPE
-#define WITHSYSTEM
+//#define WITHPIPE
+//#define WITHSYSTEM
 
 /*[13jul2005 mt]:*/
 /*With SAFESIGNAL defined, write() and read() syscalls are wrapped by
@@ -43,7 +43,7 @@ the errno checkup*/
 /*:[13jul2005 mt]*/
 
 /*[29apr2004 mt]:*/
-#define WITHEXTERNALCHANNEL
+//#define WITHEXTERNALCHANNEL
 /*
 */
 #define TRAPSIGNALS
